src/utilcode/util.cpp

```diff
@@ -1306,10 +1303,8 @@ int GetCurrentProcessCpuCount()
 
 #else // !FEATURE_PAL
 
-    SYSTEM_INFO sysInfo;
-    ::GetSystemInfo(&sysInfo);
-    cCPUs = sysInfo.dwNumberOfProcessors;
-    return sysInfo.dwNumberOfProcessors;
+    cCPUs = PAL_GetCurrentProcessCpuCount();
```

Comment:
I moved this to pal to be able to use HAVE_SCHED_GETAFFINITY. I think this can't be accessed under utilcode.